gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/PermissionService.groovy

```diff
@@ -53,4 +60,26 @@ class PermissionService {
       fiatService.sync()
     }
   }
+
+  List<String> getServiceAccounts(@SpinnakerUser User user) {
+    List<String> accounts = []
+    if (!user) {
+      log.debug("getServiceAccounts: Spinnaker user is null.")
+      return accounts
```

Comment:
I still have to have a `return` after the try/catch block, and one should try to minimize the number of potential exit points. I'll change it to `return []`